Index: app/src/main/java/com/gumibom/travelmaker/ui/main/MainActivity.kt
===================================================================
diff --git a/app/src/main/java/com/gumibom/travelmaker/ui/main/MainActivity.kt b/app/src/main/java/com/gumibom/travelmaker/ui/main/MainActivity.kt
--- a/app/src/main/java/com/gumibom/travelmaker/ui/main/MainActivity.kt	
+++ b/app/src/main/java/com/gumibom/travelmaker/ui/main/MainActivity.kt	
@@ -36,12 +36,13 @@
     fun navigationToNextFragment(){
         navController.navigate(R.id.action_mainFragment_to_mainFindMateFragment)
     }
+
     private fun initToolbar(){
         binding.toolbar.setNavigationOnClickListener {
             // Handle navigation icon press
         }
 
-        binding.toolbar.setOnMenuItemClickListener { menuItem ->
+        binding.toolbar.setOnMenuItemClickListener { menuItem->
             when (menuItem.itemId) {
                 R.id.action_notify -> {
                     Log.d(TAG, "initToolbar: Noyigiyu")
